Cookbook/Technical-Documents/LabelsInPRs.md

```diff
@@ -0,0 +1,118 @@
+# How to label a Pull Request
+
+
+We use labels in our Pull Requests to be able to categorize and prioritize our work. Some specific labels are also used to trigger our CI jobs.
+
+
+## 1. The labels
+
+At the moment we have 26 labels in our repository. They can be separated into 4 groups:
+
+- [Product verticals](#product-verticals)
+
+- [Project horizontals](#project-horizontals)
+
+- [Branding](#branding)
+
+- [Status](#status)
+
+- [Emergency](#emergency)
+
+- [Extra](#extra)
```

Comment:
I know it's outside the scope of this PR and doc, but wondering if we should actually make those groups appear as prefix or emojis in the label names?

In multiple other repos, labels have a prefix to mention the group they belong to. For example:

 - [in SwiftGen, labels are prefixed with `difficulty:` / `status:` or `type:`](https://github.com/SwiftGen/SwiftGen/labels)
 - [in CocoaPods, they're prefixed `d1:`/`d2:`/`d3:` for difficulty level, `s1:`/`s2:`/`s3:`/... for various statuses, `t1:`/`t2:`/`t3:`/... for PR type](https://github.com/CocoaPods/cocoapods/labels)...

Of course renaming the ones used by the bot would require to update the bots too, so those ones might require some care, but otherwise it might help consistency –and also help understanding of the role of those labels and have them grouped alphabetically in the list– to use a similar idea in our repo? (maybe just an emoji as prefix – although I'm wondering if some emojis are not the cause of the GitHub error which sometimes happen and make us prefer to add the labels _after_ the PR is created?). wdyt?

Reply:
This is a great idea, I actually thought about that while I was writing this documentation. But as you said, this is out of scope of this PR 😛 However, I can use this as an excuse to work on that later 🙂